### ISSUE TITLE
Newline in markdown tables

### ISSUE THREAD

**Issue by andrew-sumner** [concordion/concordion#220]
The only way to insert a line break into a markdown table is to use the &lt;br&gt; tag.  With the release of Concordion 2.0.3 to include the fix for #208, we no longer have this ability.

For example this 

```
| [response][][Outcome][] | [Message][]                                   |
| ----------------------- | --------------------------------------------- |
| Manual                  | Does not comply with:<br />Rule 1<br />Rule 2 |
```

Should render as:

| Outcome | Message |
| --- | --- |
| Manual | Does not comply with:<br />Rule 1<br />Rule 2 |

But is now rendered as:

| Outcome | Message |
| --- | --- |
| Manual | Does not comply with:&lt;br /&gt;Rule 1&lt;br /&gt;Rule 2 |

**Comment by andrew-sumner** [comment]
@nigelcharman has suggested one option could be to use some custom JavaScript to modify your output spec. It could be somewhat similar to the "View as Rendered" functionality we added to the Verify Rows Strategy spec.

The custom embed extension will also work.

Neither of these workarounds is that appealing.

**Comment by nigelcharman** [comment]
Hi Andrew, the Pegdown markdown processor is deprecated. I've replaced it with Flexmark in #272. Are you able to give this a go (using [Jitpack](https://jitpack.io/)), and let me know if it fixes this issue?

**Comment by nigelcharman** [comment]
Fixed in 3.0.0
